{
    "config": {
        "abort": {
            "already_configured": "Der Dienst ist bereits konfiguriert",
            "already_in_progress": "Der Konfigurationsablauf wird bereits ausgef\u00fchrt",
            "cannot_connect": "Verbindung fehlgeschlagen",
            "reauth_successful": "Die erneute Authentifizierung war erfolgreich"
        },
        "error": {
            "cannot_connect": "Verbindung fehlgeschlagen",
            "invalid_access_token": "Ung\u00fcltiger Zugriffs-Token"
        },
        "step": {
            "user": {
                "data": {
                    "host": "Host",
                    "port": "Port"
                }
            }
        }
    }
}